### ISSUE TITLE
Validate IETF commits/user mappings: mailarch

### ISSUE THREAD

**Issue by ronaldtse** [ietf-ribose/svn-github-convert#25]
```
rcross = rcross <rcross@amsl.com>
henrik = henrik <henrik>
mark = mark <mark@painless-security.com>
```

**Comment by ronaldtse** [comment]
## FIXED

personal/markd/2.1.12

<img width="792" alt="Screenshot 2021-09-30 at 11 37 45 AM" src="https://user-images.githubusercontent.com/11865/135383097-f628b3bc-977e-4034-aa62-19868f652517.png">

Updated Henrik's mapping in c168cf9.

## PENDING

Ryan Cross' email does not map on GitHub.

<img width="812" alt="Screenshot 2021-09-30 at 11 39 10 AM" src="https://user-images.githubusercontent.com/11865/135383227-af7ac690-c205-44e7-8e67-7653c8dd12c5.png">

**Comment by ronaldtse** [comment]
Ryan Cross' email from the datatracker is identical to this and does not resolve to a GitHub account.

**Comment by ronaldtse** [comment]
Ryan Cross is @rpcross and fixed via #37. Closing.
